src/canvas-effects/canvas-effect.js

```diff
@@ -11,6 +11,8 @@ import SequencerEffectManager from "../modules/sequencer-effect-manager.js";
 import { SequencerAboveUILayer } from "./effects-layer.js";
 import VisionSamplerShader from "../lib/filters/vision-mask-filter.js";
 import MaskFilter from "../lib/filters/mask-filter.js";
+import { AnimatedTilingSprite } from "../lib/pixi/AnimatedTilingSprite.js";
+import { SequencerSpriteManager } from "./sequencer-sprite-manager.ts";
```

Comment:
.ts? I think this is meant to be .js

Reply:
Oh no yes that is a remnant from when I started implementing some stuff in TS but then switched over to plain JS after asking in the discord. 
Strange that everything still worked, probably because I kept some of the type declarations for editor support?
Either way I am sorry and will fix the import